string[] fname
float64 latitude
float64 longitude
float64 altitude
sensor_msgs/Image image
std_msgs/Time time